src/java/com/zimbra/qa/soap/SoapTestMain.java

```diff
@@ -213,7 +213,7 @@ public static void parseArgs(String args[]) throws HarnessException {
         		{
         			String type = iterator.next();
         			
-        			if (type.equalsIgnoreCase("sanity")) {
+        			if (type.equalsIgnoreCase("sanity") && !globalProperties.getProperty("ignoreFiles").toString().equalsIgnoreCase("true")) {
```

Comment:
Lets not change this behavior in legacy.

Reply:
It doesn't affect Legacy